recipes/yacs/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "yacs" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2ae48bd89224d0d0fa73d7d084ca02f892664e4a08776b495ae569493e2d0f57
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pyyaml
+  run:
+    - python
+    - pyyaml
+
+test:
+  imports:
+    - yacs
+
+about:
+  home: https://github.com/rbgirshick/yacs
+  license: Apache 2.0
+  license_family: Apache
+  summary: 'YACS -- Yet Another Configuration System'
+  description: YACS -- Yet Another Configuration System
+  doc_url: https://github.com/rbgirshick/yacs
+  dev_url: https://github.com/rbgirshick/yacs
+
+extra:
+  recipe-maintainers:
+    - rbgirshick
```

Comment:
@rbgirshick, are you ok being a maintainer?

Reply:
@rgrinberg I am fine if you don't want. I'll remove you.

Reply:
@hadim thanks for being the maintainer.